### PR TITLE
CI GitHub action

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

File: .github/workflows/flutter-ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (filiph) [inline .github/workflows/flutter-ci.yml:1]: I don't think you need this here :)

Comment (domesticmouse) [reply .github/workflows/flutter-ci.yml]: done =)

File: .github/workflows/flutter-ci.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        channel: [stable, beta]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
```
Comment (johnpryan) [inline .github/workflows/flutter-ci.yml:22]: I *think* this is only necessary if you're building an APK, but probably doesn't hurt to include it 🤷‍♂️

Comment (domesticmouse) [reply .github/workflows/flutter-ci.yml:22]: If only I could test it before submitting it =)

**Comment by domesticmouse** [comment]
TBR @RedBrogdon 

I'm going to land this to test it. Because that's how this works, amirite?

**Pull request merged**